test/PerformanceTests/SerializationBaselineTests/FileSerializationBenchmarks.cs

```diff
@@ -1,90 +0,0 @@
-﻿using BenchmarkDotNet.Attributes;
-using System.Collections.Generic;
-using System.IO;
-
-namespace SerializationBaselineTests
-{
-    [MemoryDiagnoser]
-    public class FileSerializationBenchmarks

```

Comment:
I combined these benchmarks and the in-memory serialization benchmarks into a single class and instead used the benchmark categories feature to distinguish them